aiidalab_widgets_base/structures.py

```diff
@@ -685,30 +694,30 @@ def _pybel_opt(self, smile, steps):
         f_f.GetCoordinates(pbmol.OBMol)
         species = [chemical_symbols[atm.atomicnum] for atm in pbmol.atoms]
         positions = np.asarray([atm.coords for atm in pbmol.atoms])
-        return self.make_ase(species, positions)
+        return self.make_ase(species, positions, smiles)
 
-    def _rdkit_opt(self, smile, steps):
+    def _rdkit_opt(self, smiles, steps):
         """Optimize a molecule using force field and rdkit (needed for complex SMILES)."""
         from rdkit import Chem
         from rdkit.Chem import AllChem
 
-        smile = smile.replace("[", "").replace("]", "")
-        mol = Chem.MolFromSmiles(smile)
+        smiles = smiles.replace("[", "").replace("]", "")
+        mol = Chem.MolFromSmiles(smiles)
         mol = Chem.AddHs(mol)
 
         AllChem.EmbedMolecule(mol, maxAttempts=20, randomSeed=42)
         AllChem.UFFOptimizeMolecule(mol, maxIters=steps)
         positions = mol.GetConformer().GetPositions()
         natoms = mol.GetNumAtoms()
         species = [mol.GetAtomWithIdx(j).GetSymbol() for j in range(natoms)]
-        return self.make_ase(species, positions)
+        return self.make_ase(species, positions, smiles)
 
-    def mol_from_smiles(self, smile, steps=10000):
+    def mol_from_smiles(self, smiles, steps=10000):
```

Comment:
Please change the method name in the other places too.
```suggestion
    def _mol_from_smiles(self, smiles, steps=10000):
```